src/views/registration-js/attendees/fields/url.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * This template renders a Single Ticket content
+ * composed by Title and Description currently
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/registration-js/attendees/fields/url.php
+ *
+ * @since   TBD
+ * @version TBD
+ *
+ * @see Tribe__Tickets_Plus__Meta__Field__Url
+ *
+ */
+
```

Comment:
:no_entry_sign: Additional blank lines found at end of doc comment

<sub>hash: 648964deda610cefc6f6fb679833a7a8</sub>